ci/hera.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/bash
+export GFS_CI_ROOT=/scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT
+export GFS_CI_HOST='Hera'
```

Comment:
Is this necessary, since we already know what `Host` we are on, without which, one wouldn't know to source this file.